.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This is a file that sets preferences on your local computer. It's not needed on the repo.

* Remove this from the repo and commit/push again

* After it's no longer in your code add it to `.gitignore`
```
# inside .gitignore file
....
.vscode/
```
* And then commit/push again the updated gitignore.

Reply:
You have to remove this

Reply:
I've removed it